app/views/admin/roles/edit.html.erb

```diff
@@ -1,21 +1,34 @@
-<% page_title "Edit #{@role.name}"  %>
-<div class="row">
-  <section class="col-md-8">
-    <h1>Edit role</h1>
-    <p class="warning">Warning: changes to roles appear instantly on the live site.</p>
+<% content_for :page_title, "Edit #{@role.name}" %>
+<% content_for :title, "Edit #{@role.name}" %>
+<% content_for :title_margin_bottom, 6 %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Edit role",
```

Comment:
We've currently got 2 headings as 1 is already set here https://github.com/alphagov/whitehall/pull/7485/files#diff-3928ca9a14107c628b94c5e47f8090a5c6ae559d6bdb2a20775610f16249eb2eR2

So we can remove this one